helfeedback/settings.py

```diff
@@ -1,23 +1,93 @@
 """
 Django settings for helfeedback project.
-
-For more information on this file, see
-https://docs.djangoproject.com/en/1.11/topics/settings/
-
-For the full list of settings and their values, see
-https://docs.djangoproject.com/en/1.11/ref/settings/
 """
 
 import os
+import environ
+import raven
 
+root = environ.Path(__file__) - 2 # two steps up, gets parent of this file
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
+env = environ.Env(
+    # Currently added as prefix to session and csrf-cookies
+    INSTANCE_NAME=(str, 'helfeedback'),
+    # Currently defines *_COOKIE_PATH, do not include leading /
+    URL_PREFIX=(str, None),
+    # DEBUG creates non-obvious vulnerabilities, keep disabled in production
+    DEBUG=(bool, False),
+    TRUST_PROXY_SSL=(bool, False),
+    TRUST_PROXY_HOST=(bool, False),
+    ALLOWED_HOSTS=(list, []),
+    ADMINS=(list, []),
+    DATABASE_URL=(str, 'postgis:///helfeedback'),
+    REDIS_URL=(str, 'redis://localhost:6379/0'),
+    MEDIA_ROOT=(environ.Path(), root('media')),
+    STATIC_ROOT=(environ.Path(), root('static')),
+    MEDIA_URL=(str, '/media/'),
+    STATIC_URL=(str, '/static/'),
+    SENTRY_DSN=(str, ''),
+    SECRET_KEY=(str, ''),
+)
+# read in environment from from "config_dev.env"
+# config_dev tries to suggest that the file should only be used
+# in development. Envvars should be used in production to avoid
+# accidents with leftover files
+environ.Env.read_env('config_dev.env')
+
+
+# Config translation from environment (can be from file)
+
+DEBUG = env('DEBUG')
+ALLOWED_HOSTS = env('ALLOWED_HOSTS')
+ADMINS = env('ADMINS')
+DATABASES = {
+    'default': env.db(),
+}
+# Celery, first line kept here for closeness
+CELERY_RESULT_BACKEND = 'djcelery.backends.database:DatabaseBackend'
```

Comment:
This doesn't seem to be right:
http://docs.celeryproject.org/en/latest/django/first-steps-with-django.html#django-celery-results-using-the-django-orm-cache-as-a-result-backend

If we're going for Redis anyway, shouldn't we use it for the result backend, too?